pkg/database/realm.go

```diff
@@ -113,10 +113,10 @@ func (r *Realm) LoadRealmUsers(db *Database, includeDeleted bool) error {
 	if includeDeleted {
 		scope = db.db.Unscoped()
 	}
-	if err := scope.Model(r).Related(&r.RealmUsers, "RealmUsers").Error; err != nil {
+	if err := scope.Model(r).Preload("Realms").Preload("AdminRealms").Order("email").Related(&r.RealmUsers, "RealmUsers").Error; err != nil {
```

Comment:
Apparently the `preload` struct tag doesn't apply on nested fields, so we have to explicitly declare it.

Reply:
not following, why does this preload AdminRealms?

Reply:
It's preloading the user's adminrealms (which is what we need to do in order for `CanAdminReam` to work. I don't know _why_ it works, but I've verified the SQL is correct